docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx

```diff
@@ -55,7 +55,10 @@ If you skip the guide in Admin Console by accident, you can also pick the guide
     },
     {
       title: 'Tranditional Web App',
-      items: [{ key: 'Traditional Web', link: '/docs/recipes/integrate-logto/traditional' }],
+      items: [
+        { key: 'Next.js', link: '/docs/recipes/integrate-logto/next-js' },
+        { key: 'Traditional Web', link: '/docs/recipes/integrate-logto/traditional' },
```

Comment:
I think we should change `Traditional Web` to either `Express` or `Node.js`, or `General` in the long term

Reply:
Yes, we can take some time to think about it.